### ISSUE TITLE
Exception on sending message from client via webSocket

### ISSUE THREAD

**Issue by mandharet** [mandharet/djangoTicTacToe#1]
Exception on sending message from client via webSocket

`Exception inside application: Incoming message has no 'type' attribute`

in builtin handlers, validation method is raising above exception when type attribute is not present in MEssage

Functions to check 

venv\Lib\site-packages\channels\consumer.py#L18

```python
def get_handler_name(message):
    """
    Looks at a message, checks it has a sensible type, and returns the
    handler name for that type.
    """
    # Check message looks OK
    if "type" not in message:
        raise ValueError("Incoming message has no 'type' attribute")
    # Extract type and replace . with _
    handler_name = message["type"].replace(".", "_")
    if handler_name.startswith("_"):
        raise ValueError("Malformed type in message (leading underscore)")
    return handler_name
```
    
 steps:
 1. start the server
 2. connect with at least two clients then send message from one client
 3. on resolving received message it's raising above exception
    
    
 _Note: While debugging many a times type found as webSocket.Receive_

**Comment by mandharet** [comment]
fix with commit id d6017708e339a5326a57443dea7636fded6c083c

https://github.com/mandharet/djangoTicTacToe/blob/d6017708e339a5326a57443dea7636fded6c083c/apps/games/consumers.py#L203-L219

**Comment by mandharet** [comment]
In summary, the issue was caused by missing 'type' attribute in the message being sent via WebSocket, and it was resolved by introducing a custom handler with the required 'type' attribute and modifying the message-sending method accordingly.
